frame/assets/src/functions.rs

```diff
@@ -564,7 +564,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 			// Check admin rights.
 			if let Some(need_admin) = maybe_need_admin {
```

Comment:
Pls rewrite this in the same way as above

Reply:
Done.